src/packages/@library/icons/CommentGridIcon.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+    export let width = "24";
+    export let height = "24";
+    export let color = "grey";
+  </script>
+  
+
+
+<svg {width}
+{height} viewBox="0 0 33 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<mask id="mask0_13086_10791" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="0" y="0" width="33" height="32">
+<rect x="0.5" width="32" height="32" fill={color}/>
+</mask>
+<g mask="url(#mask0_13086_10791)">
+<path d="M8.56602 18.8665H24.766V16.1665H8.56602V18.8665ZM8.56602 14.8165H24.766V12.1165H8.56602V14.8165ZM8.56602 10.7665H24.766V8.0665H8.56602V10.7665ZM30.166 29.6665L24.766 24.2665H5.86602C5.12352 24.2665 4.48789 24.0021 3.95914 23.4734C3.43039 22.9446 3.16602 22.309 3.16602 21.5665V5.3665C3.16602 4.624 3.43039 3.98838 3.95914 3.45963C4.48789 2.93088 5.12352 2.6665 5.86602 2.6665H27.466C28.2085 2.6665 28.8441 2.93088 29.3729 3.45963C29.9016 3.98838 30.166 4.624 30.166 5.3665V29.6665ZM5.86602 21.5665H25.9135L27.466 23.0853V5.3665H5.86602V21.5665Z" fill="#3670F7"/>
+</g>
```

Comment:
format this, file name should be generic, not limited to comment, may be used at other modules also

Reply:
Resolved